src/mapd-con-es6.js

```diff
@@ -35,7 +35,7 @@ function isNodeRuntime() {
   return typeof window === "undefined"
 }
 
-class MapdCon {
+export class MapdCon {
```

Comment:
I wonder what version bump this change merits. I'd lean towards a major bump.

Reply:
Ah, I was thinking this was mostly a different way to write the same thing (as compared to `export default MapDCon`). This can be reverted to what it was before, the only thing that was giving an issue was the `export.module` line.

Reply:
Also, regarding version change I suppose we're due for a 5.5.0 bump to follow the upcoming db release.

Reply:
@jclay If you just uncomment the last line in the file so that we still have the default export, then we have both cases covered without needing a major version bump. And we can handle the version bump separately closer to the release.

Reply:
@israelvicars sounds good and done!